### PR TITLE
nautilus: mgr/progress: remove "since" from duration string

### PR DESCRIPTION
**Pull request by shyukri**
backport tracker: https://tracker.ceph.com/issues/42442

---

backport of https://github.com/ceph/ceph/pull/31007
parent tracker: https://tracker.ceph.com/issues/40976

this backport was staged using ceph-backport.sh version 15.0.0.6950
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

### CODE REVIEW

File: src/pybind/mgr/progress/module.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import threading
 import datetime
 import uuid
-
+import time
```
Comment (smithfarm) [inline src/pybind/mgr/progress/module.py]: note: in master, this import was added by https://github.com/ceph/ceph/pull/28907 - should we consider backporting that to nautilus?

Comment (dillaman) [reply src/pybind/mgr/progress/module.py]: Yes, it is now required

**Comment by smithfarm** [comment]
@shyukri Could you please cherry-pick https://github.com/ceph/ceph/pull/28907 first and then #31007 on top of it?

**Comment by dillaman** [comment]
Looking closer, it seems like we don't need to backport this original issue to nautilus. The current branch doesn't have any duration progress tracking so there isn't anything to fix. I would recommend closing the PR and invalidating the backport tracker ticket.

**Pull request closed without merging**